futures-util/src/future/empty.rs

```diff
@@ -3,9 +3,7 @@ use core::pin::Pin;
 use futures_core::future::{Future, FusedFuture};
 use futures_core::task::{Waker, Poll};
 
-/// A future which is never resolved.
-///
-/// This future can be created with the [`empty()`] function.
+/// Future for the [`empty`] combinator.
```

Comment:
Are the free functions still "combinator"s? Maybe a better term here would be "constructor"?

Reply:
-- or drop "combinator"?

Reply:
Dropping "combinator" and rephrasing slightly to

> Future produced by [`empty`].

seems ok to me.

Reply:
It may be easier to understand using "function"s and "method"s rather than "combinator"s (I do not think that functions like `join_all` is "constructor").

Reply:
That makes sense as well, so just replace "combinator" directly with "function" or "method" depending on which it is (or even go with just "function" everywhere since method is a sub-category of function).

Reply:
Checking the `std` docs they do differentiate between "function" or "method" for the iterator types, and don't mention "combinator" at all:

> This `struct` is created by the `once` function.

> This `struct` is created by the `filter` method on `Iterator`.